tensorboard/plugins/hparams/BUILD

```diff
@@ -94,6 +94,7 @@ py_binary(
     name = "hparams_demo",
     srcs = ["hparams_demo.py"],
     srcs_version = "PY2AND3",
+    visibility = ["//visibility:public"],
```

Comment:
Nix this one and you’ll have my vote. http://cl/239833315

Reply:
Done.